apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+
+```jsx live
+<CTABanner hasBackground={false}>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Optional shadow
+
+The shadow can be removed using the `hasShadow` prop on the root `<ComparisonTable>`. This will render the CTABanner component without the background shaow.
+
+```jsx live
+<CTABanner hasShadow={false}>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+#### Color customization
+
+The default colors can be customized through dedicated CSS variables.
+
+| Name                                   | Default                          |
+| :------------------------------------- | :------------------------------- |
+| `--brand-CTABanner-shadow-color-start` | `--base-color-scale-green-5-hsl` |
+| `--brand-CTABanner-shadow-color-end`   | `--base-color-scale-blue-5-hsl`  |
+
+```javascript live noinline
+const App = () => {
+  const designTokenOverrides = `
+  .custom-colors[data-color-mode='dark'] {
+
+    /*
+     * Modify the value of these tokens.
+     * Remember to apply light mode equivalents if you're enabling theme switching.
+     */
+    --brand-CTABanner-shadow-color-start: var(--base-color-scale-purple-5-hsl);
+    --brand-CTABanner-shadow-color-end: var(--base-color-scale-red-5-hsl);
+
+  }
+`
+  return (
+    <ThemeProvider colorMode="dark" className="custom-colors">
+      <style>{designTokenOverrides}</style>
+      <CTABanner>
+        <CTABanner.Heading>
+          Where the most ambitious teams build great things
+        </CTABanner.Heading>
+        <CTABanner.Description>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+          ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+          elementum.
+        </CTABanner.Description>
+        <ButtonGroup buttonSize="large">
+          <Button>Primary Action</Button>
+          <Button>Secondary Action</Button>
+        </ButtonGroup>
+      </CTABanner>
+    </ThemeProvider>
+  )
+}
+
+render(<App />)
+```
+
+## Component props
+
+### ComparisonTable <Label>Required</Label>
```

Comment:
Heads up.. Label isn't working here. You might need to import it.
![Screenshot 2022-11-30 at 12 55 40](https://user-images.githubusercontent.com/13340707/204801913-9e1ef119-ab3e-4ac9-b231-e88ebea8c18b.png)

Reply:
Unresolving as it's still broken.. 

https://primer-25ab31e23f-26139705.drafts.github.io/components/CTABanner#comparisontable-required

Also says ComparisonTable?

```suggestion
### CTABanner <Label>Required</Label>
```